portfolio/src/main/webapp/css/map.css

```diff
@@ -0,0 +1,31 @@
+#map {
+    height: 80%;
+    width:100%;
+    top: 200px;
+    position: absolute !important;
+}
+
+#map-description {
+    margin-left: 5%;
+}
+
+.marker-form-label {
+    font-weight: bold;
+    font-family: 'Bellota Text', cursive !important;
+}
+
+.marker-textbox {
+    resize: none;
+    height: 5em !important;
+    border: 1px solid black;
+}
+
+.marker-textbox:focus {
+    outline: none;
+}
+
+.marker-submit {
+    margin-top: 5%;
+    height: 30px !important;
+    line-height: 30px !important;
+}
```

Comment:
do you find these "no new line at the end of file" linter warnings useful?
I wonder how does one tweak the gihub static checks ?!
conversely, there are CSS linters on GitHub that you can integrate with your IDE

Reply:
I usually miss adding a new line when I'm working in Cloud Shell, so I think the warnings are useful. I'm not sure if linters are supported in Cloud Shell.